src/interfaces.ts

```diff
@@ -30,7 +30,7 @@ export type Iri = Url;
 /**
  * Alias to indicate where we expect to be given a URL.
  */
-export type UrlString = string;
+export type UrlString = Iri;
```

Comment:
I think one of the core discussion elements around this change was that it would require developers to use an additional wrapper around their string IRIs when using our methods, which is an overhead.